openslides/users/static/templates/users/user-list.html

```diff
@@ -1,10 +1,147 @@
-<ul>
-    <li ng-repeat="user in users">
-        <a ui-sref="users.user.detail({id: user.id })">{{ user.get_short_name() }}</a>
-        <a ui-sref="users.user.detail.update({id: user.id })">{{ gettext('Edit') }}</a>
-    </li>
-</ul>
+<h1 translate>Participants</h1>
 
-<a os-perms="users.can_manage" ui-sref="users.user.create">{{ gettext('New') }}</a>
-<span os-perms="!users.can_manage">No Permission to create</span>
-<a href="/users/print/" target="_self">PDF</a>
+<div id="submenu">
+    <div os-perms="users.can_manage" class="btn-group">
+        <a class="btn btn-primary btn-sm" ui-sref="users.user.create">
+            <i class="fa fa-user-plus fa-lg"></i>
+            New</a>
+        <a class="btn btn-primary btn-sm dropdown-toggle" data-toggle="dropdown">
+            <span class="fa fa-caret-down"></span></a>
```

Comment:
here the same. Please check this file for the right place of the tags, there are some more lines that should be fixed.